target/scripts/startup/check-stack.sh

```diff
@@ -52,3 +52,12 @@ function _check_log_level() {
     LOG_LEVEL="${DEFAULT_LOG_LEVEL}"
   fi
 }
+
+function _check_spam_prefix() {
+  # This check should be independent of ENABLE_POP3 and ENABLE_IMAP
+  if _env_var_expect_zero_or_one MOVE_SPAM_TO_JUNK \
+  && [[ ${MOVE_SPAM_TO_JUNK} -eq 0 ]] \
```

Comment:
I don't see the value in this first condition:

https://github.com/docker-mailserver/docker-mailserver/blob/47f8d50bebaa0b1f1cb85d45c1d9210d9ce80a49/target/scripts/helpers/utils.sh#L155-L162

If you want to sanitize the variables, that should be handled in `variables-stack.sh`. Centralizing it there means we can keep our expectations simple elsewhere in the scripts that follow.

It should not be required at this point to check that the ENV if `-eq 0` to proceed.

```suggestion
  if [[ ${MOVE_SPAM_TO_JUNK} -eq 0 ]] \
```